control_panel_api/management/commands/import_k8s_users.py

```diff
@@ -1,42 +1,76 @@
 import base64
 import logging
 
+from django.conf import settings
 from django.core.management.base import BaseCommand
 from django.db.utils import IntegrityError
-from kubernetes import config, client
+from kubernetes import client, config
 
 from control_panel_api.models import User
+from moj_analytics.auth0_client import Auth0, ManagementAPI, User as Auth0User
 
 logger = logging.getLogger(__name__)
 
 
+def get_auth0_users():
+    api = Auth0(
+        settings.OIDC_CLIENT_ID,
+        settings.OIDC_CLIENT_SECRET,
+        settings.OIDC_DOMAIN
+    )
+    api.access(ManagementAPI(settings.OIDC_DOMAIN))
+
+    results = api.management.get_all(Auth0User, 'connection=github')
+
+    return results
+
+
 class Command(BaseCommand):
-    help = "Import Kubernetes user-secrets data into the User.  Uses KUBECONFIG envvar or defaults to ~/.kube/config"
+    help = "Import Kubernetes user-secrets data into the User.  " \
+           "Uses KUBECONFIG envvar or defaults to ~/.kube/config"
 
     USER_SECRETS_NAME = 'user-secrets'
 
     def handle(self, *args, **options):
         config.load_kube_config()
         v1 = client.CoreV1Api()
+
         results = v1.list_secret_for_all_namespaces()
```

Comment:
I know this is @kerin's code, but `v1` and `results` are not well named. I'd suggest `kubernetes_api` and `secrets`.

Reply:
@andyhd nothing to do with me!